api/api.go

```diff
@@ -1,19 +1,27 @@
 package api
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
+	"io"
 	"net/http"
 
 	"github.com/SkynetLabs/skynet-accounts/database"
 	"github.com/SkynetLabs/skynet-accounts/email"
 	"github.com/SkynetLabs/skynet-accounts/metafetcher"
 	"gitlab.com/SkynetLabs/skyd/build"
+	"go.mongodb.org/mongo-driver/mongo"
 
 	"github.com/julienschmidt/httprouter"
 	"github.com/sirupsen/logrus"
 	"gitlab.com/NebulousLabs/errors"
-	"go.mongodb.org/mongo-driver/mongo"
+)
+
+const (
+	// dbTxnRetryCount specifies the number of times we should retry an API
+	// call in case we run into transaction errors.
+	dbTxnRetryCount = 5
```

Comment:
Do the accounts endpoints have a general timeout param?

I'm wondering if this should be a `MaxTxnRetryCount` so that if an endpoint has a timeout param, we quit early with an error instead of continuing to retry. 

I'm not sure if we have any instances where accounts endpoint now potentially taking 5x longer would cause issues.

Might be worth a follow up discussion at least, but not necessarily blocking this PR.

Reply:
discussed on discord, goal is not to keep it open longer, but to ensure we are ending the call with whichever is faster. The number of retries or a provided/default timeout.